editor.planx.uk/src/pages/Team/components/AddFlow/index.tsx

```diff
@@ -0,0 +1,144 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Dialog, { dialogClasses } from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { isAxiosError } from "axios";
+import { Form, Formik, FormikConfig } from "formik";
+import React, { useState } from "react";
+import { useNavigation } from "react-navi";
+import { AddButton } from "ui/editor/AddButton";
+
+import { useStore } from "../../../FlowEditor/lib/store";
+import { BaseFormSection } from "./BaseFormSection";
+import { CreateFromCopyFormSection } from "./CreateFromCopyFormSection";
+import { CreateFromTemplateFormSection } from "./CreateFromTemplateFormSection";
+import { CreateFlow, validationSchema } from "./types";
+
+// TODO: Standardise or share this
```

Comment:
I'll handle this in an upcoming PR - it's been commented on a few times in previous PRs that we should have a unified dialog style.

Reply:
Appreciate standardisation here - Ian's latest publishing modal styles might change some of this a bit though! Eg slightly diverges from right-aligned buttons etc